.eslintrc.cjs

```diff
@@ -23,6 +23,7 @@ module.exports = {
     "@tanstack/query/prefer-query-object-syntax": "error",
     "@tanstack/query/stable-query-client": "error",
     "react/prop-types": "off",
+    "linebreak-style": off,
```

Comment:
Good job! forgetting out this. I didn't know there were a difference between mac and windows.